### PR TITLE
fix(map): use camo to avoid too many reqs to Google Maps Static API

### PR DESCRIPTION
**Pull request by Eywek**

### CODE REVIEW

**Comment by Eywek** [comment]
we can ignore tests, this is related to moment

File: src/helpers/map.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ module.exports.register = function (Handlebars) {
       url.searchParams.append('signature', hashedSignature)
     }
 
-    return new Handlebars.SafeString(`<img src="${url.toString()}" width="${width}" />`)
+    const base64Url = typeof window === 'undefined' ? Buffer.from(url.toString()).toString('base64') : btoa(url.toString()) // eslint-disable-line
+    return new Handlebars.SafeString(`<img src="${base64Url}" width="${width}" />`)
```
Comment (vmarchaud) [inline src/helpers/map.js:49]: i can't understand how the image will be sent to the camo here ?

Comment (Eywek) [reply src/helpers/map.js:49]: oupsi

File: src/helpers/map.js [hunk 2]
```diff
@@ -45,6 +45,8 @@ module.exports.register = function (Handlebars) {
       url.searchParams.append('signature', hashedSignature)
     }
 
-    return new Handlebars.SafeString(`<img src="${url.toString()}" width="${width}" />`)
+    const base64Url = typeof window === 'undefined' ? Buffer.from(url.toString()).toString('base64') : btoa(url.toString()) // eslint-disable-line
+    const camoURL = `${data.camoURL}/forward/${encodeURIComponent(base64Url)}`
```
Comment (vmarchaud) [inline src/helpers/map.js:49]: i think we should add a fallback in case camoURL is not defined (for example in the frontend)

**Review by vmarchaud — APPROVED** [review]

**Pull request merged**